env.example

```diff
@@ -233,6 +233,9 @@ XMPP_DOMAIN=meet.jitsi
 # Internal XMPP server
 XMPP_SERVER=xmpp.meet.jitsi
 
+# Internal XMPP server c2s port
+XMPP_PORT=5222
```

Comment:
Please leave it commented out since we are providing a default

Reply:
Done